extra/linux/org.alacritty.Alacritty.appdata.xml

```diff
@@ -1,34 +1,45 @@
+<!-- SPDX-License-Identifier: MIT -->
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright 2016-2022 Joe Wilm, The Alacritty Project Contributors -->
 <component type="desktop-application">
   <id>org.alacritty.Alacritty</id>
-  <!-- Translators: The application name -->
+
+  <developer_name>Christian Duerr</developer_name>
+
   <name>Alacritty</name>
+
   <project_license>APACHE-2.0</project_license>
```

Comment:
```suggestion
  <project_license>Apache-2.0</project_license>
```

> Please note the SPDX license IDs are case-sensitive in AppStream.

Source: https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-project_license

Reply:
Yeah let's just fix it all in one PR. I'm sure @AsciiWolf doesn't mind.